packages/react-accordion/src/components/AccordionHeader/useAccordionHeader.ts

```diff
@@ -59,6 +71,7 @@ export const useAccordionHeader = (props: AccordionHeaderProps, ref: React.Ref<H
     }),
     button: {
       ...buttonShorthand,
+      'aria-disabled': forceDisabled ?? undefined,
```

Comment:
not sure why it can't be declared in `useAriaButton` so it's spread as a part of `buttonShorthand`

Reply:
it can't be used on `useARIAbutton` because `useARIAButton` will converge `aria-disabled` and `disabled` in a single property for the case of a `button` element, and that's not the desired behavior here, the desired behavior is a forced `aria-disabled`  to show for a11y that this element is disabled but focusable. there's a new property that actually solves this on `ARIAButtonShorthandProps` which is `disabledFocusable`. I'll be reimplementing this to use this behavior